Symfony/CS/Fixer/Symfony/SpacesAfterSemicolonFixer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class SpacesAfterSemicolonFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        for ($index = count($tokens) - 3; $index > 0; --$index) {
+            if (!$tokens[$index]->equals(';') || $tokens[$index + 2]->isComment() || ' ' === $tokens[$index + 1]->getContent()) {
+                continue;
+            }
+
+            if (!$tokens[$index + 1]->isWhitespace()) {
+                $tokens->insertAt($index + 1, new Token(array(T_WHITESPACE, ' ')));
+                continue;
+            }
+
+            if (false === strpos($tokens[$index + 1]->getContent(), "\n")) {
```

Comment:
doesn't seem to work, in this case I'm looking for whitespaces that are:
- not a single space (https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1577#discussion_r47026072)  
- do not contain a line break

I tried;

```php
if (!$tokens[$index + 1]->isWhitespace(array('whitespaces' => " \n"))) {
```